### PR TITLE
SPCTR-3608: Fix - FAQ resolved faq content default open issue with <p> and <br> tag.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

[FAQ] For the shortcode that has p and br tag in the content and we used that shortcode in the FAQ answer, that FAQ box always open by default.
https://brainstormforce.atlassian.net/browse/SPCTR-3608

### Screenshots

https://d.pr/v/lwlPfX

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Container - Resolved style settings background overlay glictes.
 * Fix: Post Grid - Resolved an issue wherein the post grid block with the featured image border radius is not applied and image overflows.
 * Fix: Infobox - Resolved an issue where the page reloads upon clicking Infobox with the call-to-action set as complete box.
+* fix: FAQ - Resolved an issue where the answer tab is opened by default if content has <p> or <br> tag.
```
Comment (AbhijitNage123) [inline README.md:223]: First letter of fix should be capital

File: src/blocks/faq-child/save.js [hunk 2]
```diff
@@ -25,7 +25,9 @@ export default function save( props ) {
 					{ 'accordion' === layout && faqRenderIcon() }
 					<RichText.Content tagName={ headingTag } value={ question } className="uagb-question" />
 				</div>
-				<RichText.Content className="uagb-faq-content" tagName="p" value={ answer } />
+				<div className="uagb-faq-content">
```
Comment (AbhijitNage123) [inline src/blocks/faq-child/save.js:28]: I hope, You have tested all styling and JS related part  ( such as Opening, clossing, Keep first tab open, Keep both tab closed by default in General ) related to this changes. @rajmohanreddy774

Comment (rajmohanreddy774) [reply src/blocks/faq-child/save.js:28]: Testing is done for the above mentioned things.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**